configuration/configuring-zed.md

```diff
@@ -181,19 +182,97 @@ To override settings for a language, add an entry for that languages name to the
 
 For a complete list of languages you can override settings for see [Supported Languages](../languages/support-languages.md).
 
+## Font Features
+
+Zed supports a subset of OpenType features that can be enabled or disabled for a given buffer or terminal font:
+
+```json
```

Comment:
@as-cii is there something wrong with this code block? It's rendered weirdly on GitHub and syntax highlighting is broken on GitBook:

<img width="719" alt="CleanShot 2023-03-22 at 12 59 17" src="https://user-images.githubusercontent.com/28818/226883574-41a80a81-fd00-468d-ae5c-5c31615f829d.png">

Reply:
I guess that's not valid JSON because I am using comments as well as types. I changed it to be rendered as TypeScript, seems to look better now. Thanks for the heads-up!